### ISSUE TITLE
Training a model from scratch?

### ISSUE THREAD

**Issue by ddofer** [CyberZHG/keras-bert#189]
1. Does the package support training a model from scratch (tokenizer and all)? (Both MLM pretraining, and later finetuning)
2. Could you provide an example of the code you used for processing the data for the MLM? I see there's a generator for it that expects sentence pairs, but it's unclear to me how they should be structured (As there aren't labels there for the NSP), or how to efficiently structure it. The code you used for this would be awesome to have in the examples!
Thanks!
